core-bundle/contao/templates/twig/backend/data_container/table/menu/filter.html.twig

```diff
@@ -9,8 +9,6 @@
             options: filter.options,
             active: filter.active,
             title: filter.placeholder,
-            select_wrapper_attributes: attrs().set('data-contao--filter-target', 'filter'),
```

Comment:
If this was removed, line 6 in the parent template (`.mergeWith(select_attributes|default)`) should be removed as well as it is only used for the filter.